### PR TITLE
Fix #3248 appointment search bs4

### LINKED ISSUE #3248: Appointment Search in Calendar Needs BS4
### Describe the bug

When you click search in calendar, the search for appointments area hasn't gotten any BS4 improvements.

### To Reproduce
https://prnt.sc/rmwvq5
image
https://prnt.sc/rmwuyg
image

### PR DESCRIPTION
**Pull request by stu01509**
Fixes #3248

#### Short description of what this resolves:
Added the Bootstrap style.
Removed outdated syntax.

image

image

image

### CODE REVIEW

**Comment by adad20** [comment]
hi @stu01509 What if we align the button to the center? It will look nice.

**Comment by bradymiller** [comment]
Regarding best positioning of buttons, this has been a bit contentious in past. The light theme places them to side and the colors themes centers them via some css trickery. At that time, I looked up some research on this (yes, folks have systematically looked at this :) ) and it looked like to the side was quicker for eyes and clicking. For the styling guide, we should probably come up with a standard though.

File: interface/main/calendar/modules/PostCalendar/pntemplates/default/user/ajax_search.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [-include file="$TPL_NAME/views/header.html"-]
 
 [-* we want to include our stylesheet for this view*-]
-<link rel="stylesheet" href="[-$TPL_STYLE_PATH-]/ajax_search.css" type="text/css">
+<!-- <link rel="stylesheet" href="[-$TPL_STYLE_PATH-]/ajax_search.css" type="text/css"> -->
```
Comment (bradymiller) [inline interface/main/calendar/modules/PostCalendar/pntemplates/default/user/ajax_search.html]: better to remove than comment out. and would remove the ajax_search.css file also.

**Comment by bradymiller** [comment]
hi @stu01509 , Looks really nice. The next step down the road will be making the form responsive but that can be done in future. Only noted one issue (and then get to remove a legacy file which is always exciting :) ); after that is addressed, will bring this into codebase.

**Comment by stu01509** [comment]
Hi @bradymiller 

Please review it again, Thank you :)

File: interface/main/calendar/modules/PostCalendar/pntemplates/default/user/ajax_search.html [hunk 2]
```diff
@@ -2,7 +2,6 @@
 [-include file="$TPL_NAME/views/header.html"-]
 
 [-* we want to include our stylesheet for this view*-]
-<link rel="stylesheet" href="[-$TPL_STYLE_PATH-]/ajax_search.css" type="text/css">
```
Comment (bradymiller) [inline interface/main/calendar/modules/PostCalendar/pntemplates/default/user/ajax_search.html]: fare thee well interface/main/calendar/modules/PostCalendar/pntemplates/default/style/ajax_search.css, you will NOT be missed :)

**Comment by bradymiller** [comment]
hi @stu01509 , Code looks good and bringing into codebase. Thanks for the improvements! -brady

**Pull request merged**